cmd/cape/cmd/root.go

```diff
@@ -183,22 +186,29 @@ func initConfig() {
 	}
 }
 
-func formatterFromString(s string) log.Formatter {
+func setRenderer(cmd *cobra.Command, s string) {
+	ctx := cmd.Context()
+
+	var r render.Renderer
+	r = render.Print{}
 	switch s {
 	case "json":
-		return &log.JSONFormatter{}
-	case "log":
-		return &log.TextFormatter{}
-	case "log_plain":
-		return &log.TextFormatter{
-			DisableColors: true,
-			FullTimestamp: true,
-		}
-	case "plain":
+		r = &render.JSON{}
+	case "human":
```

Comment:
why do we need this case if it just fallsthrough to the next case (default)?

Reply:
It's nice to have a name for the setting, rather than have the default be something that can't be explicitly set, which is why I have the string case there. In this case the behavior will be the same as the default case where no setting or an invalid setting was specified.

Other options I considered were making them both explicit, and making the default case an error condition rather than just ignoring the input and falling back to human-readable.